inputgraph/loader.go

```diff
@@ -188,6 +224,12 @@ func (l *loader) handleCopySrc(ctx context.Context, src string, mustExist bool)
 	return nil
 }
 
+var sha1RE = regexp.MustCompile("^[0-9a-f]{40}$")
+
+func validRemoteTarget(t domain.Target) bool {
```

Comment:
Minor suggestion: Maybe rename this to `supportedRemoteTarget`? The unsupported ones are still valid.